doc/tdshim_spec.md

```diff
@@ -752,15 +752,21 @@ MRTD is extended by two operations:
 the chunk’s content.
 
 The expected flow is:
-1. Search TD Metadata from metadata location and get TDVF_DESCRIPTOR.
-2. In TDVF_DESCRIPTOR, get TDVF_SECTION from index 0 to (NumberOfSectionEntry -
+1. Initialize a SHA384 hash context. Load TDVF image and parse.
+2. Search TD Metadata from metadata location and get TDVF_DESCRIPTOR.
+3. In TDVF_DESCRIPTOR, get TDVF_SECTION from index 0 to (NumberOfSectionEntry -
    1).
-3. In TDVF_SECTION, if PAGE.AUG attribute is clear (PAGE.ADD is used), extend
-   MRTD with the THD.MEM.PAGE.ADD as described above, from the lowest GPA to the
-   highest GPA, with 4K granularity.
-4. In TDVF_SECTION, if MR.EXTEND attribute is set, extend MRTD with the
-   TDH.MR.EXTEND as described above, from the lowest GPA to the highest GPA,
-   with 256 bytes granularity.
+4. For every TDVF_SECTION in the order of index, Walk through the TDVF_SECTION
+   in 1 page (4096 bytes) granularity.
+    1. For every page,
```

Comment:
I suggestion:
 * For every TDVF_SECTION in TDVF_DESCRIPTOR, from index 0 to (NumberOfSectionEntry -
   1)
   * For every page (4K granularity) in TDVF_SECTION, from lowest GPA to  highest GPA
     * If PAGE.AUT ...
     * If MR.EXTEND ...